e2e-polybft/cardanofw/test-skyline-bridge.go

```diff
@@ -0,0 +1,77 @@
+package cardanofw
```

Comment:
No need two have two separate files for skyline and reactor. Just depending on config (`IsReactor`) execute one of constructor methods. Everything else is the same